.github/workflows/cla-check.yml

```diff
@@ -0,0 +1,31 @@
+name: "CLA Check"
+
+on:
+    pull_request_target:
+        branches: ["main"]
+
+jobs:
+    cla-check:
+        #if: github.event.pull_request.user.login != 'renovate[bot]'
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v5
+              with:
+                  repository: neo-technology/whitelist-check
+                  token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
```

Comment:
Doesn't look like this secret is defined in the repo?

Reply:
Good catch, the name is very similar and didn't noticed the slight difference